include/oauth.php

```diff
@@ -116,7 +116,7 @@ function new_access_token($token, $consumer, $verifier = null) {
 
 
 	if (!is_null($ret) && $uverifier!==false){
-		del_config("oauth", $verifier);
+		Config::delele("oauth", $verifier);
```

Comment:
Typo: It should be `delete`, not `delele`.

Reply:
hahah oops

Reply:
And yes, I did check for load, nothing to change. I'll go through addons and update there as well.